docker.gradle

```diff
@@ -25,6 +25,12 @@ buildscript {
 	}
 }
 
+if (project.hasProperty("tag")) {
```

Comment:
Please:
1. Use `global-settings.gradle` to set the variable defaults; see that file for examples.
2. Update the variable names as described in my notes on `README.md`

Reply:
Updated